learn/experimental/puffin_reports.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Puffin reports — Meilisearch documentation
+description: Puffin is an instrumentation profiler for Rust applications. Use this experimental feature when trying to diagnose performance issues during indexing.
+---
+
+# Puffin reports
+
+Puffin is an instrumentation profiler for Rust applications. Use this feature when trying to diagnose performance issues during indexing. Avoid enabling Puffin reports in production, as it might affect the performance of other Meilisearch operations.
```

Comment:
```suggestion
Puffin is an instrumentation profiler for Rust applications. Use this feature when trying to diagnose performance issues during indexing. Avoid enabling Puffin reports in production, as it might affect the performance and memory usage of other Meilisearch operations.
```